lib/dotcom_web/components/system_status/commuter_rail_status.ex

```diff
@@ -0,0 +1,265 @@
+        "text-black no-underline font-normal",
+        "border-t-[1px] border-gray-lightest",
+        "min-h-12 text-md md:text-lg"
+      ]}
+      href={@row.url}
+      data-test="status-row"
+    >
+      <div class="grid items-center grid-cols-[min-content_min-content_auto] items-center grow">
+        <div class="flex items-center pl-1 pr-2 min-w-64 md:min-w-72">
+          <span>{@row.name}</span>
+        </div>
+
+        <div class="pr-2 flex items-center">
+          <.status_icon status={:normal} />
+        </div>
+
+        <div class="grow flex items-center">
+          <.status_label status={:normal} />
+        </div>
+      </div>
+      <div class="self-stretch flex items-center">
+        <.icon name="chevron-right" class="h-3 w-2 fill-gray-dark ml-3 mr-2 shrink-0" />
+      </div>
+    </a>
+    """
+  end
+
+  # For cases where we have alerts, we have to show the first alert along with route information
+  # and then show subsequent rows without the route information.
+  defp row(%{row: %{alert_counts: alert_counts}} = assigns) do
+    [first | rest] = combine_alert_counts(alert_counts)
+    assigns = assigns |> assign(:first, first) |> assign(:rest, rest)
+
+    ~H"""
+    <a
+      class={[
+        "flex items-center py-2",
+        "hover:bg-brand-primary-lightest cursor-pointer group/row",
+        "text-black no-underline font-normal",
+        "border-t-[1px] border-gray-lightest",
+        "min-h-12 text-md md:text-lg"
+      ]}
+      href={@row.url}
+      data-test="status-row"
+    >
+      <div class="grid items-center grid-cols-[min-content_min-content_auto] items-center grow">
+        <div class="flex items-center pl-1 pr-2 min-w-64 md:min-w-72 border-bottom-[1px] border-black">
+          <span>{@row.name}</span>
+        </div>
+
+        <div class="pr-2 flex items-center">
+          <.status_icon status={elem(@first, 0)} />
+        </div>
+
+        <div class="grow flex items-center font-bold">
+          {elem(@first, 1)}
+        </div>
+      </div>
+      <div class="self-stretch flex items-center">
+        <.icon name="chevron-right" class="h-3 w-2 fill-gray-dark ml-3 mr-2 shrink-0" />
+      </div>
+    </a>
+    <a
+      :for={row <- @rest}
+      class={[
+        "flex items-center py-2",
+        "hover:bg-brand-primary-lightest cursor-pointer group/row",
+        "text-black no-underline font-normal",
+        "border-t-[1px] border-gray-lightest",
+        "min-h-12"
+      ]}
+      href={@row.url}
+      data-test="status-row"
+    >
+      <div class="grid items-center grid-cols-[min-content_min-content_auto] items-center grow">
+        <div class="flex items-top pl-1 pr-2 min-w-64 md:min-w-72 min-h-full border-t-[1px] -mt-[25px] border-white">
+        </div>
+
+        <div class="pr-2 flex items-center">
+          <.status_icon status={elem(row, 0)} />
+        </div>
+
+        <div class="grow flex items-center font-bold">
```

Comment:
```suggestion
        <div class="grow flex items-center font-bold text-md md:text-lg">
```

It's important to make sure this matches the styles above. As it currently stands, the first entry inherits a `text-md md:text-lg` from its parent, which makes it 18px on desktop, but this one doesn't (I'm guessing because the route name only appears on the first row when there are multiple statuses worth reporting), so this winds up as 16px.

![Screenshot 2025-05-15 at 2 42 35 PM](https://github.com/user-attachments/assets/74043de8-7147-49aa-98e4-254bdafd5437)